tests/commands/sgaTests.js

```diff
@@ -0,0 +1,126 @@
+var sga = require("__buttercup/classes/commands/command.sga.js");
+
+var fakeGroup;
+
+module.exports = {
+  setUp: function(cb) {
+    this.command = new sga();
+    (cb)();
```

Comment:
Ah I've figured it out - Atom on Windows for me was set to only indent 2 spaces. Atom on Ubuntu was set to 4 spaces. I'll fix this in about 25 minutes.